manifests/kibana.pp

```diff
@@ -43,13 +50,32 @@
     try_sleep => 3,
   }
 
+  exec {'Removing old Wazuh Kibana App Plugin if exists...':
+    path    => '/usr/bin',
+    command => "sudo rm /usr/share/kibana/plugins/wazuh -rf",
+    notify  => Service[$kibana_service],
+  }
+
   exec {'Installing Wazuh App...':
     path    => '/usr/bin',
     command => "sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp-${kibana_app_version}.zip",
     creates => '/usr/share/kibana/plugins/wazuh/package.json',
     notify  => Service[$kibana_service],
   }
 
+  exec {'Removing .wazuh index...':
+    path    => '/usr/bin',
+    command => "curl -s -XDELETE -sL -w '%{http_code}' -I 'http://${kibana_elasticsearch_ip}:${kibana_elasticsearch_port}/.wazuh' -o /dev/null",
```

Comment:
Are you using the HTTP status code in the curl request? `-w '%{http_code}'` 

Looks like it could be omitted.

Reply:
I checked this and `-w '%{http_code}'` does not provide extra info even when executing Puppet agent with `--debug` flag which was the reason for using it.

Removed in 3a56f2269d584b33f985c7a0b9a4965241f0b134